index.js

```diff
@@ -286,8 +286,8 @@ class GithubScm extends Scm {
         try {
             const pullRequestInfo = await this.getPrInfo({ scmUri, token, prNum });
 
-            if (pullRequestInfo.mergeable) {
-                return { success: true, pullRequestInfo };
+            if (pullRequestInfo.mergeable !== null) {
```

Comment:
Does it ever return `undefined` ?

Reply:
No.
But `pullRequestInfo.mergeable` may be `undefined` in exception case.
I will add `undefined` check.
Thanks.